modules/nw-multus-bridge-object.adoc

```diff
@@ -82,25 +93,24 @@ plugin:
 |Optional: Enables mac spoof check, limiting the traffic originating from the container to the mac address of the interface. The default value is `false`.
 |====
 
+ifndef::microshift[]
 [NOTE]
 ====
 The VLAN parameter configures the VLAN tag on the host end of the `veth` and also enables the `vlan_filtering` feature on the bridge interface.
 ====
 
 [NOTE]
 ====
-To configure uplink for a L2 network you need to allow the vlan on the uplink interface by using the following command:
+To configure uplink for an L2 network, you must allow the VLAN on the uplink interface by using the following command:
```

Comment:
This might also be beyond the scope of this PR. But, is there a better word to use than _allow_? I keep asking you must allow the VLAN to do what? 
```suggestion
To configure uplink for an L2 network, you must add the VLAN to the uplink interface by using the following command:
```

Reply:
```suggestion
To configure an uplink for an L2 network, you must allow the VLAN on the uplink interface by using the following command:
```

Reply:
"Allow" is standard for networking, yeah. You "enable" connections, and "allow" the network traffic.